e2e/test/helpers/templates/FaultInjection.cs

```diff
@@ -243,15 +213,6 @@ public static bool FaultShouldDisconnect(string faultType)
                 }
                 deviceClient.Dispose();
                 await testDevice.RemoveDeviceAsync().ConfigureAwait(false);
-
-                faultInjectionDuration.Stop();
-
-                TimeSpan timeToFinishFaultInjection = duration.Subtract(faultInjectionDuration.Elapsed);
-                if (timeToFinishFaultInjection > TimeSpan.Zero)
-                {
-                    logger.Trace($"{nameof(FaultInjection)}: Waiting {timeToFinishFaultInjection} to ensure that FaultInjection duration passed.");
-                    await Task.Delay(timeToFinishFaultInjection).ConfigureAwait(false);
```

Comment:
the hub-wide faults might still need this delay.

Reply:
Which are hub wide?

Reply:
iirc that would be quota exceeded, throttled and bad auth.